_data/projects/mediawiki.yml

```diff
@@ -0,0 +1,12 @@
+name: MediaWiki
+desc: The collaborative editing software that runs Wikipedia
+site: https://www.mediawiki.org
+tags:
+- mediawiki
+- wiki
+- web
+- cms
+- php
+upforgrabs:
+  name: Easy
+  link: https://phabricator.wikimedia.org/tag/easy/
```

Comment:
This is fine - "Open tasks" is available in the sidebar, it's not like it's hard to find...